module/users/helper.js

```diff
@@ -874,11 +874,9 @@ module.exports = class UsersHelper {
         let programsResult = userRelatedPrograms.map(id => {
           return userRelatedProgramsData.find(data => data._id.toString() === id.toString());
         });
-        // to remove null values  from program result as private programs will be not listed
-        programsResult = programsResult.filter(element => {return element !== null && element !== undefined})
```

Comment:
Any reason for removing this line? If it wasn't required, why was it added?

Reply:
Previously i added isAPrivateProgram in the ProgramDocument function but later on, shifted to getUserJoinedProgramDetailsWithPreviousProfiles that why this line is not required. hence removed This was added by me only in this PR